proxy/http/Http1ClientSession.h

```diff
@@ -219,6 +220,9 @@ class Http1ClientSession : public ProxyClientSession
 
   HttpServerSession *bound_ss;
 
+  int released_transactions;
+  bool in_destroy;
```

Comment:
ProxyClientSession has `in_destroy` and it's a protected (not private) member. Why do we need this?